packages/core/test/actor.test.ts

```diff
@@ -1105,4 +1106,75 @@ describe('actors', () => {
       })
       .start();
   });
+
+  // https://github.com/statelyai/xstate/issues/2507
+  it('should not attempt to clean up children before stopping', () => {
```

Comment:
This IMHO describes a vague internal implementation detail. The actual user-like story would sound something like:
"should not crash on sync child completion during own setup"